pom.xml

```diff
@@ -100,6 +100,13 @@
             <version>3.2</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+        	<groupId>org.apache.sling</groupId>
+        	<artifactId>org.apache.sling.testing.sling-mock</artifactId>
```

Comment:
this introduces a cyclic dependency between the api artifact and the sling-mock artifact (also only in test scope). not sure if this will bring us trouble later.

Reply:
I'm not clear on how this could be an issue with the sling-mock being in the test scope